diff.py

```diff
@@ -2579,6 +2601,29 @@ def process(dump: str, config: Config) -> List[Line]:
         if not row:
             continue
 
+        if config.diff_function_symbols == True:
+            # If diffing function symbols is enabled
+            # Check if the line contains a function label
+            # If so, create a line to diff for the label
+
+            # Regex the current row to check for "OFFSET <SYM>:"
+            funcNameMatch = re.match(r"[0-9A-Fa-f]+ <(.+)>:", row)
+            
+            if funcNameMatch:
+                # Add the function symbol to the diff output
+                funcSym = funcNameMatch.groups()[0] + ":"
+                output.append(
+                    Line(
+                        mnemonic=funcSym, # this is needed for score changes
+                        diff_row=funcSym, # this is needed to be set for highlighting
+                        original=funcSym, # this is the value that is printed
+                        normalized_original="...",
+                        scorable_line="...",
+                    )
+                )
+                continue
+        
+        # Filter out function label lines "OFFSET <SYM>:"
         if re.match(r"^[0-9a-f]+ <.*>:$", row):
```

Comment:
This is the same check, right? Can you change the logic so it's
```
if a:
  if b:
    ...
  continue
```
instead of
```
if b:
  if a:
    ...
    continue
if a:
  continue
```
?